docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -236,14 +236,11 @@ test('Take Screenshot test', async t => {
 });
 ```
 
-## Work with iframes
+## Work With Iframes
 
 A TestCafe test's [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#windows) is limited to the main window or an `<iframe>`. To use an `<iframe>` in your test,
-switch the context from the main window to this `<iframe>`.
-If several `<iframes>` are used in your test, you need to switch between them.
-limited to either the main window or an `<iframe>`. To use an `<iframe>` in your test,
-you need to switch the context from the main window to this `<iframe>` (and then probably back).
-Likewise, if several `<iframes>` are involved in your test, you will have to switch between them.
+switch the context from the main window to this `<iframe>` (and then probably back).
```

Comment:
What do you mean by "and then probably back"? For example, do you mean:

`switch the context from the main window to the `<iframe>`. (You may need to switch back to the main window later.)`

Reply:
Exactly, thanks

Reply:
Thanks, please apply my suggestion.

Reply:
done